src/templates/containers/CommunityTemplates.scss

```diff
@@ -99,6 +99,7 @@
 }
 
 .community-templates--resources-table {
+  margin-left: 1.4em;
```

Comment:
This is so that the resource table aligns with the header text instead of the caret end.